docs/source/sections/understanding/plug_and_play.rst

```diff
@@ -0,0 +1,4 @@
+Plug-and-Play
+=============
+
+aioauth was designed to be as flexible as possible to allow developers to choose their own server framework, as well as database provider. Since aioauth is written as an asynchronous module it would be to the advantage of the developer to write their applications asynchronously.
```

Comment:
great explanation!

Reply:
> @synchronizing wow very impressive! many thanks for contributing!

My pleasure on the contribution. I see the huge potential this library has, and I think documentation this early on would help flourishing the repo down the line. I have future plans to eventually helping out implementing OpenID Connect as well. 

> great explanation!

Thank you! I'm writing things as I go, so if you feel like anything is out-of-line, please don't hesitate to call it out/change it. I'll be working on this further come the next month- I got a huge deadline dropped on my table yesterday.